ipn/serve.go

```diff
@@ -150,19 +156,36 @@ func (sc *ServeConfig) IsTCPForwardingOnPort(port uint16) bool {
 	if sc == nil || sc.TCP[port] == nil {
 		return false
 	}
-	return !sc.TCP[port].HTTPS
+	return !sc.IsServingWeb(port)
 }
 
 // IsServingWeb checks if ServeConfig is currently serving
-// Web/HTTPS on the given port.
+// Web (HTTP/HTTPS) on the given port.
 // This is exclusive of TCPForwarding.
 func (sc *ServeConfig) IsServingWeb(port uint16) bool {
+	return sc.IsServingHTTP(port) || sc.IsServingHTTPS(port)
+}
+
+// IsServingHTTPS checks if ServeConfig is currently serving
```

Comment:
for bool methods, the conventional wording is "reports whether"

Reply:
Updated these and all the others in this file. Spring cleaning.